test_env_vars.yaml

```diff
@@ -126,41 +126,43 @@ pip_packages:
   # expected_files: Expected source files <relative_path>: <file_URL>
   # expected_deps_files: Expected dependencies files (empty)
   # dependencies: expected dependencies from the Cachito response
-  # package: expected package from the Cachito response
+  # packages: expected packages from the Cachito response
   # purl: PURL of the package
   without_deps:
     repo: https://github.com/cachito-testing/cachito-pip-without-deps.git
     ref: fb07c8492432631d557aba15e1715af9cef9c844
+    pkg_managers: ["pip"]
     expected_files:
       setup.cfg: https://raw.githubusercontent.com/cachito-testing/cachito-pip-without-deps/master/setup.cfg
       setup.py: https://raw.githubusercontent.com/cachito-testing/cachito-pip-without-deps/master/setup.py
     expected_deps_files: null
     dependencies: []
-    package:
-      dependencies: []
-      name: "cachito-pip-empty"
-      type: "pip"
-      version: "1.0.0"
+    packages:
+      - dependencies: []
+        name: "cachito-pip-empty"
+        type: "pip"
+        version: "1.0.0"
     purl: "pkg:pypi/cachito-pip-empty@1.0.0"
   # pip package with dependencies in requirements.txt
   # repo: The URL for the upstream git repository
   # ref: A git reference at the given git repository
   # expected_files: Expected source files <relative_path>: <file_URL>
   # expected_deps_files: Expected dependencies files <relative_path>: <archive_URL>
```

Comment:
is this still correct?

Reply:
I believe so, the test is passing with these variables.

Reply:
changed that